### PR TITLE
Feature/cors

### PR DESCRIPTION
**Pull request by davidvdw02**
cors gefixt 

we accepteren nu heel veel van http://localhost:4200 met deze code.

### CODE REVIEW

**Review by DaaromThomas — APPROVED** [review]
Approved

File: src/main/java/com/hsleiden/vdlelie/config/CorsConfig.java
Comment (ronp2001) [inline src/main/java/com/hsleiden/vdlelie/config/CorsConfig.java:1]: no allowed method for patch?

**Review by ronp2001 — CHANGES_REQUESTED** [review]
see comment on CorsConfig

File: src/main/java/com/hsleiden/vdlelie/config/CorsConfig.java
Comment (davidvdw02) [reply src/main/java/com/hsleiden/vdlelie/config/CorsConfig.java:1]: added patch

**Review by ronp2001 — APPROVED** [review]
Looks good to me

**Pull request merged**